atomic_defi_design/qml/Screens/NewUser.qml

```diff
@@ -34,6 +34,19 @@ SetupPage {
         return current_mnemonic.split(" ")
     }
 
+    function getNumberTh(num) {
+        switch (num) {
+            case 1:
+                return 'st'
```

Comment:
I think you need to use `qsTr()` on those strings since thoses nb suffixes are not the same across diferent languages.

Reply:
fixed